src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster-review.component.html

```diff
@@ -12,9 +12,8 @@
         <tr>
           <td i18n
               class="bold">Storage Capacity</td>
-          <td><span i18n
-                    *ngIf="filteredDevices && capacity">Number of devices: {{ filteredDevices.length }}. Raw capacity:
-            {{ capacity | dimlessBinary }}.</span></td>
+          <td><span i18n>Number of devices: {{ totalDevices }}. Raw capacity:
```

Comment:
LGTM! Just 1 thing abt this: Ok we show `Storage capacity: .... Raw capacity` , well tht does't makes sense IMO, storage capacity itself here represents the total  capacity disks provides(may not be consumed fully) which is actually raw capacity itself. 
So maybe we can hv just 2 different rows for `Devices` & `Storage Capacity`.

Reply:
@avanthakkar I don't think its a good idea to separate Devices out of the Storage Capacity field. Because these device count is specific to the Storage Capacity (to know how many no. of storage devices are there) and placing them in a separate field will make it look meaningless. So the best thing will be to keep it under the Storage Capacity heading and then split the `No .of Devices` and 'Raw Capacity` into two columns.